### PR TITLE
For #95: Implemented FloorListView for listing floors

### PR DESCRIPTION
**Pull request by paulodamaso**
For #95: Implemented `FloorListView` for listing floors
- Implemented `list_floors()` through `FloorListView`, which extends `ListView`
- Created test assuring that the floor information is showed correctly on screen
- Left puzzles for finishing the implementation of Floors endpoints (create, edit and delete)

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#402](https://github.com/timelesslounge/timelessis/issues/402) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#402](https://github.com/timelesslounge/timelessis/issues/402) is assigned to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/4ce99993-9095-40a9-840f-e79f7efa599e); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: tests/view/floors/test_floors.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    print(response.data)
+    assert response.status_code == HTTPStatus.OK
+    assert b"<p class=\"description\">1st Floor</p>" in response.data
+    assert b"<p class=\"description\">2nd Floor</p>" in response.data
+    assert b"<p class=\"description\">3rd Floor</p>" in response.data
+    assert b"<p class=\"description\">4th Floor</p>" in response.data
+    FloorListView.model = Floor
+
+
+@pytest.mark.skip(reason="/floors/create not implemented")
```
Comment (ammaratef45) [inline tests/view/floors/test_floors.py]: @paulodamaso You should leave a puzzle for these tests to be implemented?

**Comment by ammaratef45** [comment]
@paulodamaso I left a comment and there are problems with codeclimate and unit tests

**Comment by paulodamaso** [comment]
@ammaratef45 The puzzles are in https://github.com/timelesslounge/timelessis/pull/402/files#diff-5e9e363e91901ff0e41b9139677c5539 please take a look

File: tests/view/floors/test_floors.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from http import HTTPStatus
+
+from tests.view.floors.mock_floors import FloorMock
+from timeless.restaurants.models import Floor
+from timeless.restaurants.floors.views import FloorListView
+
+""" Tests for floors.
+@todo #95:30min Continue implementation of
```
Comment (ammaratef45) [inline tests/view/floors/test_floors.py]: @paulodamaso This puzzle body is very small

**Comment by ammaratef45** [comment]
@paulodamaso The tests still failing

File: tests/view/floors/test_floors.py (hunk 2 above)
Comment (paulodamaso) [reply tests/view/floors/test_floors.py]: @ammaratef45 Sorry this was commited by mistake.

**Comment by paulodamaso** [comment]
@ammaratef45 The tst was outdated, marked as skipped, please take a look now

**Comment by ammaratef45** [comment]
@paulodamaso you forgot to delete the puzzle which committed by mistake

File: tests/view/floors/test_floors.py (hunk 2 above)
Comment (paulodamaso) [reply tests/view/floors/test_floors.py]: @ammaratef45 Sorry, rough morning here

**Comment by paulodamaso** [comment]
@ammaratef45 All good now, thanks

**Comment by ammaratef45** [comment]
@paulodamaso Great.

**Comment by ammaratef45** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@emilianodellacasa Merged master, please take a look

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 15 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#402](https://github.com/timelesslounge/timelessis/issues/402) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@ammaratef45 According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by ammaratef45** [comment]
> @ammaratef45 According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
> 
> > The code reviewer found at least three problems in the code.
> > Comments were mostly about design problems, not cosmetic issues.
> 
> Only one issue was found during code review.
> Please confirm that you'll try to find at least three major problems while future reviews.

@ypshenychka confirm

**Comment by ypshenychka** [comment]
@ammaratef45 thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**